index.html

```diff
@@ -22,17 +22,15 @@
 
       <nav class="nav-bar">
         <div class="left-navbar">
-          <a href="#" class="logo">Muhammad Zunair khan</a>
-          <div class="wrap">
-            <a href="#">
-              <div class="menu"></div>
-              <div class="menu"></div>
-              <div class="menu"></div>
-            </a>
+          <a id="logo" href="#" class="logo">Muhammad Zunair khan</a>
```

Comment:
- [ ] When clicking on any link on the mobile menu the menu should disappear, kindly make sure to implement this so the website behaves as expected